worlds/witness/player_logic.py

```diff
@@ -387,6 +394,41 @@ def handle_postgame(self, world: "WitnessWorld") -> List[List[str]]:
 
         return postgame_adjustments
 
+    def handle_panelhunt_postgame(self, world: "WitnessWorld") -> List[List[str]]:
+        postgame_adjustments = []
+
+        # Make some quick references to some options
+        panel_hunt_postgame = world.options.panel_hunt_postgame
+        chal_lasers = world.options.challenge_lasers
+
+        disable_mountain_lasers = (
+            panel_hunt_postgame == "disable_mountain_lasers_locations"
+            or panel_hunt_postgame == "disable_anything_locked_by_lasers"
+        )
+
+        disable_challenge_lasers = (
+            panel_hunt_postgame == "disable_challenge_lasers_locations"
+            or panel_hunt_postgame == "disable_anything_locked_by_lasers"
+        )
+
+        if disable_mountain_lasers:
+            self.DISABLE_EVERYTHING_BEHIND.add("0x09F7F")  # Short box
+            self.HUNT_ENTITIES.add("0x09F7F")
+            self.COMPLETELY_DISABLED_ENTITIES.add("0x3D9A9")  # Elevator Start
+
+            # If mountain lasers are disabled, and challenge lasers > 7, the box will need to be rotated
+            if chal_lasers > 7:
+                postgame_adjustments.append([
+                    "Requirement Changes:",
+                    "0xFFF00 - 11 Lasers - True",
+                ])
+        if disable_challenge_lasers:
```

Comment:
You usually put an extra line in places like this
```suggestion

        if disable_challenge_lasers:
```